### DISCUSSION TITLE
Configuration export/import system

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#10024] [Feature Requests]
Making changes on a live backend admin should not be recommended. Working on the Directus backend which is serving a live site can be risky for all sorts of reasons. If the backend faces issues the site suffers. Exporting and importing configuration changes between a Directus installation in different environments, such as Local and Production, allows admins to make and verify changes before going live. This workflow of a local -> stage -> live helps in a more robust dev ops setup

**Comment by ghost** [comment]
This can also help in 
1. Sharing components/pages/features
2. Grouping features in profiles

**Comment by rijkvanzanten** [comment]
This exists! You can use the `directus schema snapshot` and `directus schema apply` commands to save / import the schema configuration to / from a file 👍🏻 See https://docs.directus.io/reference/cli/#migrate-schema-to-a-different-environment

**Reply by ApolloRoboto** [reply]
The cli seems to have been deprecated, how do you achieve this today?

**Reply by rijkvanzanten** [reply]
Those cli commands still exist 🙂 You could also use the API endpoints to achieve the same: https://directus.io/docs/api/schema

**Reply by ApolloRoboto** [reply]
Thank you!
